www/on/%platform/associate.spt

```diff
@@ -64,13 +64,9 @@ elif action == 'connect':
     try:
         user.participant.take_over((platform.name, account.user_id))
     except NeedConfirmation, obstacles:
-        # XXX Eep! Internal redirect! Really?!
-        # XXX In this case, we can get away with reusing the same dispatch_result. When upstreaming
-        #     we should probably generalize by making a new DispatchResult.
-        request.internally_redirected_from = dispatch_result.match
-        resource = resources.get(website, website.www_root+'/on/confirm.html.spt')
-        state.update(locals())
-        raise resource.respond(state)
```

Comment:
@Changaco You're removing the internal redirect on principle, because it's a "mess"?